.github/workflows/next-sync.yml

```diff
@@ -19,6 +19,14 @@ jobs:
         with:
           node-version: 14
 
+      - name: Prepare Lingui-Bot git account
+        uses: oleksiyrudenko/gha-git-credentials@v1
+        with:
+          name: 'Lingui Bot'
+          email: 'linguibot@gmail.com'
+          actor: 'lingui-bot'
+          token: '${{ secrets.GH_BOT_PWD }}'
```

Comment:
Hey @semoal, sorry to bother you, but could you please explain to me again what's the benefit of using password over a token?

I've checked the `gha-git-credentials` repository and they use it in their example:

```yml
    # publish to a branch in current repo using GITHUB_TOKEN and other default settings
    - uses: oleksiyrudenko/gha-git-credentials@v1
      with:
        token: '${{ secrets.GITHUB_TOKEN }}'
```

Reply:
oh, didn't see that, i only saw the first case that was the case of remote repo. Sorry mate was my fault

Reply:
I change it now

Reply:
I have one question about that because if we use the GITHUB_TOKEN, will do the login correctly to the linguibot account?

Reply:
I changed it to github token, looks if we are in the same repo/org works perfectly with GITHUB_TOKEN and we'll override the committer.

Reply:
Nevermind my last sentence, that's wrong I've tried on my lingui fork and the pull request is created by the github action bot account, it's not using the lingui git account.

Reply:
we can create a personal access token on lingui bot account, to use that instead of the password I guess. what do you think?

Reply:
I've already created a personal access token, but I've add it as `GH_TOKEN` and not `GITHUB_TOKEN`. I've just checked that I can't add `GITHUB_TOKEN`, I guess it's reserved for GitHub Actions account.

Reply:
Perfect so, I'll change the name asap

Reply:
Done Tom